smarsy/bs_helper.py

```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup
+
+
+class BSHelper(object):
+    """
+     The help class for BeautifulSoup library
+     """
+    def __init__(self, html):
+        self.html = html
```

Comment:
I do not see why do we want to have `html` in the constructor. In such case that logically mean that you need to create new instance of the object if you want to change the value of `html`